docs/realm.js

```diff
@@ -288,13 +288,44 @@ Realm.defaultPath;
  *   object types in this Realm. **Required** when first creating a Realm at this `path`.
  * @property {number} [schemaVersion] - **Required** (and must be incremented) after
  *   changing the `schema`.
- * @property {Object} [sync] - Sync configuration parameters with the following 
+ * @property {Object} [sync] - Sync configuration parameters with the following
  *   child properties:
  *   - `user` - A `User` object obtained by calling `Realm.Sync.User.login`
  *   - `url` - A `string` which contains a valid Realm Sync url
  *   - `error` - A callback function which is called in error situations
  *   - `validate_ssl` - Indicating if SSL certificates must be validated
  *   - `ssl_trust_certificate_path` - A path where to find trusted SSL certificates
+ *   - `ssl_verify_callback` - A callback function used to accept or reject the server's
+ *      SSL certificate. ssl_verify_callback is called with an object of type
+ *      {
+ *        serverAddress: String
+ *        serverPort: Number
+ *        pemCertificate: String
+ *        preverifyOk: Number
```

Comment:
why not boolean value instead of Number. by the comments below seems like a boolean will suffice

Reply:
for the preverifyOk

Reply:
also why not change this name to openSSLAccepted, acceptedbyOpenSSL,or something like mustVerify

Reply:
Yes, I was very close to making it a boolean. I just kept the OpenSSL API.  
I agree it is better with a boolean. 
I have changed it to boolean and called it acceptedByOpenSSL.